Userland/Games/Minesweeper/Field.cpp

```diff
@@ -341,8 +341,10 @@ void Field::on_square_clicked_impl(Square& square, bool should_flood_fill)
         return;
     if (square.is_considering)
         return;
-    if (!m_timer->is_active())
+    if (!m_timer->is_active()) {
+        m_timer->on_timeout();
```

Comment:
`on_timeout()` is never set now, so this should be unnecessary.

Reply:
`on_timeout()` is still set. It’s just being done by the Timer constructor now.

https://github.com/SerenityOS/serenity/blob/977aa8131055ef4185c6f4fbde6b1412f063f542/Userland/Libraries/LibCore/Timer.cpp#L17-L20

Reply:
:yakthonk: It's a bit janky that you need to manually call a timer's callback, but fair enough! It's not related to what you did here.